src/app/ubs/ubs-user/ubs-user-profile-page/ubs-profile-change-password-pop-up/ubs-profile-change-password-pop-up.component.html

```diff
@@ -5,20 +5,6 @@ <h2 mat-dialog-title class="title">{{ 'ubs-client-profile.change-password' | tra
   </div>
   <form [formGroup]="formConfig" (ngSubmit)="onSubmit()">
     <div *ngIf="hasPassword">
-      <mat-form-field *ngIf="{ isHidden: true } as passwordInput">
-        <mat-label>{{ 'ubs-client-profile.old-password' | translate }}</mat-label>
-        <input matInput [type]="passwordInput.isHidden ? 'password' : 'text'" formControlName="currentPassword" />
```

Comment:
Remove corresponding control in formgroup.

Reply:
Yeah,thaks :)